libsolidity/ast/Types.h

```diff
@@ -527,7 +527,7 @@ class FixedBytesType: public Type
 	int numBytes() const { return m_bytes; }
```

Comment:
This should also return `unsigned`.

Reply:
And then of course, at every place this is used, it should be changed to `unsigned`. At some point, if the changes go to far, then of course we can convert to int again, but let's change a bit more for now.

Reply:
And in many cases those places have assertions for negative numbers which will not be needed after this change.